packages/account/src/Containers/account.jsx

```diff
@@ -109,25 +109,8 @@ const PageOverlayWrapper = ({
     );
 };
 
-const Account = ({
-    active_account_landing_company,
-    currency,
-    history,
-    is_from_derivgo,
-    is_logged_in,
-    is_logging_in,
-    is_pre_appstore,
-    is_risky_client,
-    is_virtual,
-    is_visible,
-    location,
-    logout,
-    platform,
-    routeBackInApp,
-    routes,
-    should_allow_authentication,
-    toggleAccount,
-}) => {
+const Account = observer(({ history, location, routes }) => {
+    const { client, common, ui } = useStore();
```

Comment:
```suggestion
    const { client, common, ui } = useStore();
    const {
        is_pre_appstore,
        is_virtual,
        landing_company_shortcode: active_account_landing_company,
        is_risky_client,
        should_allow_authentication,
        currency,
        is_logged_in,
        is_logging_in,
        logout,
    } = client;
    const { routeBackInApp, is_from_derivgo, platform } = common;
    const { toggleAccountSettings: toggleAccount, is_account_settings_visible: is_visible } = ui;
```
@yaswanth-deriv Let's do object destructuring in all the components🙏🏻
This helps to not make changes in the component itself so we don't change the history for lines, And also fixes issues when we use the variables as dependencies like in the `useEffect` 🙇🏻

Reply:
@farzin-deriv ..Thank you very much for ur comments.I addressed all of it.